microcosm/api.py

```diff
@@ -1,9 +1,15 @@
+from microcosm.config.validation import required
 from microcosm.decorators import binding, defaults
+from microcosm.loaders import load_each, load_from_dict, load_from_environ
 from microcosm.object_graph import create_object_graph
 
 
 __all__ = [
     binding,
     create_object_graph,
     defaults,
+    load_each,
```

Comment:
These all feel like they deserve to be part of the official API based on our usage patterns.